src/ts_generator.py

```diff
@@ -0,0 +1,364 @@
+from dataclasses import dataclass
```

Comment:
We need to add copyright headers in all files (see example in andromede-modeling-prototype).